src/components/play/brick/baseComponents/CountDown.tsx

```diff
@@ -4,12 +4,15 @@ import "./BrickCounter.scss";
 import CounterDown from "./CounterDown";
 import { BrickLengthEnum } from "model/brick";
 import Clock from "./Clock";
+import { Moment } from 'moment';
 const moment = require("moment");
```

Comment:
Can't these both be simplified into one import statement?
Like `import { Moment }, * as moment from 'moment';`

Reply:
not sure will check

Reply:
Don't think we really should be using a require statement anyway if it's not totally necessary.